server/vulnerabilities/nvd/cve.go

```diff
@@ -404,66 +404,69 @@ func checkCVEs(
 						// No such vendor in the Vulnerability dictionary
 						continue
 					}
-					cacheHits := cache.Get([]*wfn.Attributes{CPEItem.GetMeta()})
-					for _, matches := range cacheHits {
-						if len(matches.CPEs) == 0 {
-							continue
-						}
 
-						if rule, ok := knownNVDBugRules.FindMatch(
-							matches.CVE.ID(),
-						); ok {
-							if !rule.CPEMatches(CPEItem.GetMeta()) {
+					cpeItemsWithAliases := expandCPEAliases(CPEItem.GetMeta())
+					for _, cpeItem := range cpeItemsWithAliases {
+						cacheHits := cache.Get([]*wfn.Attributes{cpeItem})
```

Comment:
I did try that but it was going to be more complex because I had to determine which of the provided aliases caused the hit (for getting the resolvedInVersion, etc.).